ci/test-locally.sh

```diff
@@ -24,9 +23,25 @@ cleanup() {
     set -x
 }
 trap cleanup EXIT
-
 trap "exit 24" INT TERM
 
+python3 -m pip install --user -U ../batch
+
+if [[ $CLOUD_SQL_PROXY -eq 1 ]]; then
+    export CLOUD_SQL_CONFIG_PATH=`pwd`/batch-secrets/batch-test-cloud-sql-config.json
+    connection_name=$(jq -r '.connection_name' $CLOUD_SQL_CONFIG_PATH)
+    port=$(jq -r '.port' $CLOUD_SQL_CONFIG_PATH)
+    ./cloud_sql_proxy -instances=$connection_name=tcp:$port &
+    proxy_pid=$!
+else
+    export CLOUD_SQL_CONFIG_PATH=/batch-secrets/batch-test-cloud-sql-config.json
+fi
+
+export BATCH_SERVER_URL=http://127.0.0.1:5001
```

Comment:
But locally, we're using a fresh batch server based off of current branch?

Reply:
both PR tests and local tests ultimately call `test-locally.sh`, see the last line or `test-in-cluster.sh`